src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--fri">
+      <div class="calendar__item calendar__item--1"></div>
+
+      <div class="calendar__item calendar__item--2"></div>
+
+      <div class="calendar__item calendar__item--3"></div>
+
+      <div class="calendar__item calendar__item--4"></div>
+
+      <div class="calendar__item calendar__item--5"></div>
+
+      <div class="calendar__item calendar__item--6"></div>
+
+      <div class="calendar__item calendar__item--7"></div>
+
+      <div class="calendar__item calendar__item--8"></div>
+
+      <div class="calendar__item calendar__item--9"></div>
```

Comment:
Remove number modifiler
Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)
It sets the last day to show (use [nth-child](https://css-tricks.com/how-nth-child-works/))

```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length-31">
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
      <div class="calendar__item"></div>
```